[2025-04-05T08:56:41.791+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: alpha_vantage_pipeline.load_to_bq manual__2025-04-05T08:56:21.946265+00:00 [queued]>
[2025-04-05T08:56:41.799+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: alpha_vantage_pipeline.load_to_bq manual__2025-04-05T08:56:21.946265+00:00 [queued]>
[2025-04-05T08:56:41.799+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-05T08:56:41.799+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-04-05T08:56:41.799+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-05T08:56:41.811+0000] {taskinstance.py:1300} INFO - Executing <Task(GCSToBigQueryOperator): load_to_bq> on 2025-04-05 08:56:21.946265+00:00
[2025-04-05T08:56:41.824+0000] {standard_task_runner.py:55} INFO - Started process 335 to run task
[2025-04-05T08:56:41.831+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'alpha_vantage_pipeline', 'load_to_bq', 'manual__2025-04-05T08:56:21.946265+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/alpha_vantage_pipeline.py', '--cfg-path', '/tmp/tmpjakwf9uu']
[2025-04-05T08:56:41.833+0000] {standard_task_runner.py:83} INFO - Job 80: Subtask load_to_bq
[2025-04-05T08:56:41.913+0000] {task_command.py:388} INFO - Running <TaskInstance: alpha_vantage_pipeline.load_to_bq manual__2025-04-05T08:56:21.946265+00:00 [running]> on host f3d042561a09
[2025-04-05T08:56:41.974+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=alpha_vantage_pipeline
AIRFLOW_CTX_TASK_ID=load_to_bq
AIRFLOW_CTX_EXECUTION_DATE=2025-04-05T08:56:21.946265+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-04-05T08:56:21.946265+00:00
[2025-04-05T08:56:41.984+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-04-05T08:56:41.984+0000] {gcs_to_bigquery.py:377} INFO - Using existing BigQuery table for storing data...
[2025-04-05T08:56:41.985+0000] {credentials_provider.py:323} INFO - Getting connection using `google.auth.default()` since no key file is defined for hook.
[2025-04-05T08:56:42.043+0000] {gcs_to_bigquery.py:381} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'finflow-455108', 'datasetId': 'my_dataset', 'tableId': 'final_table'}, 'sourceFormat': 'PARQUET', 'sourceUris': ['gs://finflow-455108-data-lake/intermediate/alpha_vantage/*.parquet'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False}}
[2025-04-05T08:56:42.044+0000] {bigquery.py:1542} INFO - Inserting job ***_alpha_vantage_pipeline_load_to_bq_2025_04_05T08_56_21_946265_00_00_8d8771d74c2fd807bf7e405df62ad7b6
[2025-04-05T08:56:43.570+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 440, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 137, in result
    raise self._exception
google.api_core.exceptions.NotFound: 404 Not found: Uris gs://finflow-455108-data-lake/intermediate/alpha_vantage/*.parquet
[2025-04-05T08:56:43.584+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=alpha_vantage_pipeline, task_id=load_to_bq, execution_date=20250405T085621, start_date=20250405T085641, end_date=20250405T085643
[2025-04-05T08:56:43.597+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 80 for task load_to_bq (404 Not found: Uris gs://finflow-455108-data-lake/intermediate/alpha_vantage/*.parquet; 335)
[2025-04-05T08:56:43.609+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-04-05T08:56:43.632+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
